cli/cmd/vuln_host.go

```diff
@@ -242,25 +254,35 @@ To list the CVEs found in the hosts of your environment run:
 			}
 
 			rows := hostVulnHostsTable(response.Hosts)
-			// if the user wants to show only online or
-			// offline hosts, show a friendly message
-			if len(rows) == 0 {
-				if vulCmdState.Online {
-					cli.OutputHuman("There are no online hosts.\n")
-				}
-				if vulCmdState.Offline {
-					cli.OutputHuman("There are no offline hosts.\n")
+			if vulCmdState.Csv {
+				cli.OutputHuman(
+					renderAsCSV(
+						[]string{"Machine ID", "Hostname", "External IP", "Internal IP",
+							"Os/Arch", "Provider", "Instance ID", "Vulnerabilities", "Status"},
+						rows,
+					),
+				)
+			} else {
```

Comment:
nitpick: we prefer to avoid `if .. else .. ` statements when possible, this `if` could end with `return nil`